google_shopping.js

```diff
@@ -1,5 +1,87 @@
 var data = require("./products.json")
 
-// Write your solutions below
+ 
+
+//Question 1:
+console.log(data["items"].length);
+
```

Comment:
This is one way to see how many items there are in total. To view how many items have a 'kind' of 'shopping#product', you can perform a loop as follows:

var numberOfShoppingProducts = null;
```for (var i = 0; i < data["items"].length; i += 1) {```
	```if (data["items"][i].kind == 'shopping#product') {```
	```numberOfShoppingProducts++```
	```};```
```};```